build-concourse/step-definitions.ts

```diff
@@ -68,7 +69,7 @@ export const GIT_PDF_STEPS = [
 export const CLI_GIT_WEB_STEPS = [
     get('step-fetch'),
     get('step-prebake'),
-    get('step-bake'),
+    get('step-bake-web'),
```

Comment:
Should all-epub use this as well? I saw that docx also uses this new step: epub would then be the only build based on web outputs that does not use step-bake-web.

Reply:
oops, no, it's more like docx shouldn't use the new step 🤦

Reply:
Then you will want to replace
```typescript
export const CLI_GIT_GDOC_STEPS = [
    ...CLI_GIT_WEB_STEPS,
    get('step-docx'),
]
```
with
```typescript
export const CLI_GIT_GDOC_STEPS = [
    get('step-fetch'),
    get('step-prebake'),
    get('step-bake'),
    get('step-postbake'),
    get('step-disassemble'),
    get('step-jsonify'),
    get('step-docx'),
]
```

So that the docx steps no longer inherit `step-bake-web` from `CLI_GIT_WEB_STEPS`.